configs/ext/crypto_config_profile_medium.h

```diff
@@ -619,11 +611,6 @@
 // We expect TF-M to pick this up soon
 #define MBEDTLS_BLOCK_CIPHER_NO_DECRYPT
 
-/* CCM is the only cipher/AEAD enabled in TF-M configuration files, but it
- * does not need CIPHER_C to be enabled, so we can disable it in order
- * to reduce code size further. */
-#undef MBEDTLS_CIPHER_C
```

Comment:
MBEDTLS_CIPHER_C appears to have been removed from config, but it is still used in `tf_psa_crypto_check_config.h`

Is that intentional?